src/chrome/content/rules/MERY.jp.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```

Comment:
Why are they problematic?
They all WFM.

Please also add www. (redirects to plain ^)